app/views/posts/index.html.erb

```diff
@@ -1 +1,32 @@
-<h1>All Posts will be shows here</h1>
\ No newline at end of file
+<div class="container">
+
+
+
+<% if @posts.empty? %> 
+  <div>This user has not published any post yet!</div>
+<% else %>
+
+<% @posts.each_with_index do |post, index| %>
+
+  <div class="overflow-hidden mt-5">
+    <div><%= link_to "Post  ##{index + 1}", user_post_path(post.author, post) %></div>
+    <br>
+    <div><%= post.text %></div>
+    <div class="float-end">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></div>
+  </div>
+
```

Comment:
- [ ] In the posts index page you should also show the user details (photo, name, and number of posts) as on the wireframe. Please make sure to fix this issue. (You can use the same code as you're using on the users index, and show page).

![image](https://github.com/Aqurds/coding-thought/assets/83803414/5bddfee6-e3b1-4548-a6f9-7778e34a9288)

Reply:
Added user data. Thanks